composer.json

```diff
@@ -12,11 +12,17 @@
         }
     ],
     "require": {
-        "roundcube/plugin-installer": ">=0.1.6"
+        "php": "^5.4 || ^7.0 || ^8.0",
+        "roundcube/plugin-installer": "^0.1.6"
```

Comment:
I don't think this is equal to `>=0.1.6`, as it's a leading 0 version, see https://getcomposer.org/doc/articles/versions.md#caret-version-range-. So this could block the newer 0.2 and 0.3 versions, which we are completly fine with.

Reply:
I can assure you this is equal
I will prevent major versions and let any equal or above version install

Reply:
Oh, sorry then, I am mistaken 🤦🏻 
I did not know there was an exception for 0.x patterns 
I will change that in my next push

Reply:
Done, sorry for this mistake

Reply:
No worries, that's what reviews are for! Still at work atm, but will probably have a look later today.